Add Recruitment FAQs (#139)
alan-turing-institute/REG-handbook: content/docs/join_us/recruitment_FAQs.md

The discussion will likely touch on the performance of the algorithm in terms of e.g. speed, accuracy or range of applicability, but our goal is to evaluate your understanding of the algorithm, analysis or software and its implementation rather than to assess you on the performance of your particular implementation.

### Should the presentation be verbal only or should I produce some materials alongside it - would the committee prefer presentation directly from an IDE, or using a presentation software?

You should be able to show and (potentially) run the code (this could be using Jupyter notebooks, Rmarkdown, a mix of an IDE and the command line).
You are welcome to include any other material that could be useful for supporting your presentation.

### What will the problem solving aspect of the second interview involve? Will I be asked about particular data science or machine learning approaches? Will I need to write code or explain the details of an algorithm's implementation?

In the second interview we expect you to discuss a question related to the analysis of an example dataset.

> All information required to answer the question will be given to you in beginning of the interview. <b>No preparation or specific technical knowledge is expected or required</b>. The goal of this exercise is not to test your understanding of any particular algorithm or data science method, but rather to assess your general approach to tackling a data science question.

Reviewer: ```suggestion
All information required to answer the question will be given to you in beginning of the interview.
**No preparation or specific technical knowledge is expected or required.**
The goal of this exercise is not to test your understanding of any particular algorithm or data science method, but rather to assess your general approach to tackling a data science question.
```

This exercise will not require you to write any code or explain the details of any algorithm.
The exercise isn't assessed on whether or not you know the "right" answer to a particular data science question, but on the process you go through to understand the question and the data and your reasoning for the approaches you suggest.
Taking some time to think through the question is fine, as is asking questions to the interviewers.
Consider them as subject matter experts coming to you to answer a question using their data.

### Is there anything in particular I should prepare for the problem solving interview?

Nothing at all.
The best thing you could do to be prepared is to eat well, take some long walks far away from the computer and get a good night's sleep.

## Offer

### What are the reasons of your job offer? Is the salary negotiable?

Fairness and transparency are key values for us and our goal is to avoid inequity by ensuring we are paying people the same salary for the same level of contribution to the team.

> <b>Our offers are not negotiable</b> and set by considering the level and applicability of each candidate’s experience relative to their future peers already working in the team.

Reviewer: ```suggestion
**Our offers are not negotiable** and set by considering the level and applicability of each candidate’s experience relative to their future peers already working in the team.
```

We have annual pay reviews, where we actively review the performance and development of each team member against the expectations of their position within their seniority band and make salary adjustments where these expectations are being exceeded.
Additionally, the team is growing and there will be opportunities for people to progress in their career.

Becoming a member of REG will also bring the following benefits:

- Flexible working: we are currently trialling being a “remote-first” group for the entire decision-making process, but as the office is now fully re-opened members are free to choose how often they come into the office.

- The opportunity to work and collaborate on cutting-edge data science projects with high-profile institutions, but with the benefits of working in a full-time position (better than a post-doc, better than many data science roles in industry).

> - Strong input into which projects you work on. We value providing the experience of learning something new in a collaborative setting, and prioritise staffing people onto projects they would like to work on rather than maximising the fit to people's prior experience. Our group is made up of people with a wide range of skills and experiences and we generally find we can put together project teams with a suitable mix of skills from those who are enthusiastic about each project.
> 
> - Formal support for professional development via learning and development plans each team member develops with their line manager during the probation process, annual objective setting and regular 1-2-1s. These learning plans feed into the projects people are allocated to and the roles and responsibilities they undertake within these.
> 
> - Support to team members with their development outside of projects, providing 20% time that is split evenly between self-directed development and getting involved in one of the group's or institute's service areas. We support team members to attend conferences, workshops and courses, and there are many seminar series and interest groups at the Turing that team members are encouraged to get involved with, including the team’s own weekly [tech talks]({{< relref "docs/regular_events/lunchtime_tech_talks.md" >}}). We also support team members making contributions to the wider ecosystem as part of their 20% time. Examples include contributing to external [Open Source Hacksessions]({{< relref "docs/regular_events/open_source_hacksessions.md" >}}) and contributing to community groups like the [Society for Research Software Engineering](https://society-rse.org/).
> 
> - The opportunity to get involved non-project work through one of the team's service areas, some of which support the running of the team and some of which support the wider research community at the institute (for instance Recruitment, Planning and finance, Training, EDI, Communications, Research computing support, Research programme liaison). Some service areas require significant support and will have core team members who have chosen these service areas instead of a full project workload, but all team members have a 10% allocation to volunteer for a service area of their choice.
> 
> - The opportunity to contribute to how the group works and grows. We run the team in an open, consultative and collaborative manner and almost all of our decision making on how we organise ourselves and develop the team is done in the open on our group GitHub with the opportunity for anyone in the team to contribute.

Reviewer: ```suggestion
- Strong input into which projects you work on.
  We value providing the experience of learning something new in a collaborative setting, and prioritise staffing people onto projects they would like to work on rather than maximising the fit to people's prior experience.
  Our group is made up of people with a wide range of skills and experiences and we generally find we can put together project teams with a suitable mix of skills from those who are enthusiastic about each project.

- Formal support for professional development via learning and development plans each team member develops with their line manager during the probation process, annual objective setting and regular 1-2-1s.
  These learning plans feed into the projects people are allocated to and the roles and responsibilities they undertake within these.

- Support to team members with their development outside of projects, providing 20% time that is split evenly between self-directed development and getting involved in one of the group's or institute's service areas.
  We support team members to attend conferences, workshops and courses, and there are many seminar series and interest groups at the Turing that team members are encouraged to get involved with, including the team’s own weekly [tech talks]({{< relref "docs/regular_events/lunchtime_tech_talks.md" >}}).
  We also support team members making contributions to the wider ecosystem as part of their 20% time.
  Examples include contributing to external [Open Source Hacksessions]({{< relref "docs/regular_events/open_source_hacksessions.md" >}}) and contributing to community groups like the [Society for Research Software Engineering]({{< relref "docs/onboarding/society_of_research_software_engineering.md" >}}).

- The opportunity to get involved non-project work through one of the team's service areas, some of which support the running of the team and some of which support the wider research community at the institute (for instance Recruitment, Planning and finance, Training, EDI, Communications, Research computing support, Research programme liaison).
  Some service areas require significant support and will have core team members who have chosen these service areas instead of a full project workload, but all team members have a 10% allocation to volunteer for a service area of their choice.

- The opportunity to contribute to how the group works and grows.
  We run the team in an open, consultative and collaborative manner and almost all of our decision making on how we organise ourselves and develop the team is done in the open on our group GitHub with the opportunity for anyone in the team to contribute.
```

  These learning plans feed into the projects people are allocated to and the roles and responsibilities they undertake within these.

- Support to team members with their development outside of projects, providing 20% time that is split evenly between self-directed development and getting involved in one of the group's or institute's service areas.
  We support team members to attend conferences, workshops and courses, and there are many seminar series and interest groups at the Turing that team members are encouraged to get involved with, including the team’s own weekly [tech talks]({{< relref "docs/regular_events/lunchtime_tech_talks.md" >}}).
  We also support team members making contributions to the wider ecosystem as part of their 20% time.
  Examples include contributing to external [Open Source Hacksessions]({{< relref "docs/regular_events/open_source_hacksessions.md" >}}) and contributing to community groups like the [Society for Research Software Engineering]({{< relref "docs/onboarding/society_of_research_software_engineering.md" >}}).

> - The opportunity to get involved non-project work through one of the team's service areas, some of which support the running of the team and some of which support the wider research community at the institute (for instance Recruitment, Planning and finance, Training, EDI, Communications, Research computing support, Research programme liaison).

Reviewer 2: ```suggestion
- The opportunity to get involved in non-project work through one of the team's service areas, some of which support the running of the team and some of which support the wider research community at the institute (for instance Recruitment, Planning and finance, Training, EDI, Communications, Research computing support, Research programme liaison).
```
